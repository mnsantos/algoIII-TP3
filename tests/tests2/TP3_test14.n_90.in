90 1248
1 2
1 8
1 7
1 3
8 10
2 3
1 4
9 10
1 16
1 17
4 7
5 8
1 9
1 5
1 10
5 7
2 18
6 11
2 9
10 11
8 14
1 11
7 13
1 15
1 14
2 5
3 16
2 10
3 17
7 8
1 18
7 15
7 11
4 13
3 9
1 6
1 12
2 11
3 15
3 13
3 12
88 90
87 90
85 90
85 86
84 90
84 89
84 88
83 89
83 88
83 84
82 89
82 87
82 85
81 88
81 87
81 84
80 88
80 82
79 84
79 83
78 80
77 88
77 87
77 86
77 85
77 84
77 78
76 85
76 84
76 80
75 89
75 86
75 82
74 90
74 89
74 86
74 85
74 84
74 83
74 77
74 76
73 90
73 88
73 85
73 84
73 81
73 75
73 74
72 90
72 87
72 86
72 85
72 84
72 83
72 78
72 77
72 75
72 73
71 81
70 90
70 88
70 84
70 83
70 79
70 78
70 76
70 75
70 72
69 90
69 89
69 88
69 86
69 82
69 73
68 87
68 86
68 84
68 83
68 81
68 80
68 78
68 75
68 72
68 71
67 90
67 88
67 82
67 72
67 70
66 88
66 84
66 83
66 82
66 81
66 80
66 75
66 69
65 88
65 73
65 72
65 70
64 88
64 87
64 83
64 82
64 78
64 74
64 72
64 65
63 89
63 83
63 73
63 69
62 80
62 79
62 77
62 76
62 74
62 72
62 71
61 90
61 87
61 77
61 75
61 72
61 71
61 70
61 65
61 63
61 62
60 89
60 87
60 86
60 84
60 82
60 81
60 75
60 71
60 69
60 68
60 61
59 88
59 83
59 82
59 81
59 74
59 70
59 69
59 67
59 65
59 64
58 90
58 88
58 84
58 82
58 81
58 78
58 75
58 71
58 64
57 85
57 84
57 81
57 78
57 75
57 68
57 66
57 65
57 64
57 60
57 58
56 90
56 86
56 85
56 79
56 77
56 75
56 72
56 68
56 67
56 66
56 64
56 59
56 58
55 88
55 87
55 85
55 77
55 73
55 70
55 66
55 65
55 64
55 63
55 60
55 56
54 90
54 89
54 88
54 83
54 79
54 75
54 73
54 72
54 70
54 68
54 64
54 62
54 61
54 58
53 89
53 88
53 87
53 84
53 79
53 78
53 75
53 72
53 71
53 69
53 65
53 63
53 59
53 58
53 57
52 89
52 87
52 80
52 79
52 76
52 73
52 71
52 69
52 62
52 60
52 57
52 53
51 90
51 89
51 88
51 85
51 83
51 75
51 73
51 71
51 69
51 67
51 61
51 56
51 52
50 90
50 83
50 76
50 73
50 72
50 64
50 51
49 90
49 87
49 78
49 72
49 71
49 69
49 68
49 66
49 65
49 62
49 59
49 54
49 51
48 88
48 75
48 74
48 70
48 68
48 67
48 65
48 57
47 89
47 77
47 72
47 70
47 63
47 55
46 90
46 88
46 84
46 83
46 78
46 76
46 75
46 72
46 60
46 56
46 54
46 52
46 49
46 48
45 87
45 80
45 78
45 77
45 75
45 73
45 65
45 62
45 50
45 47
44 88
44 86
44 85
44 82
44 81
44 78
44 72
44 65
44 63
44 59
44 54
44 52
44 48
43 88
43 85
43 82
43 75
43 72
43 71
43 69
43 67
43 65
43 60
43 59
43 56
43 48
43 45
43 44
42 90
42 84
42 73
42 71
42 65
42 63
42 62
42 59
42 58
42 57
42 56
42 49
41 87
41 84
41 80
41 69
41 66
41 60
41 58
41 57
41 55
41 53
41 48
41 45
41 42
40 83
40 82
40 81
40 68
40 64
40 61
40 57
40 55
40 48
40 45
40 44
40 41
39 90
39 88
39 87
39 77
39 73
39 62
39 59
39 56
39 51
39 50
39 48
39 41
39 40
38 90
38 89
38 88
38 87
38 84
38 81
38 80
38 75
38 74
38 71
38 65
38 62
38 52
38 50
38 49
38 41
38 40
38 39
37 86
37 79
37 78
37 77
37 73
37 70
37 69
37 67
37 65
37 56
37 55
37 49
37 46
36 90
36 86
36 83
36 80
36 74
36 69
36 68
36 67
36 62
36 61
36 57
36 56
36 53
36 49
36 48
36 44
36 43
36 41
36 39
36 37
35 88
35 86
35 85
35 83
35 81
35 80
35 76
35 75
35 70
35 65
35 64
35 61
35 58
35 52
35 48
35 47
35 40
34 84
34 83
34 82
34 81
34 78
34 72
34 71
34 67
34 49
34 46
34 43
34 42
34 38
33 90
33 82
33 81
33 78
33 74
33 69
33 67
33 61
33 60
33 55
33 46
33 43
13 62
13 61
13 60
13 59
13 58
13 55
13 54
13 51
13 47
13 46
13 45
13 41
13 37
13 35
13 32
13 29
13 27
13 25
13 15
12 89
12 87
12 84
12 79
12 77
12 76
12 72
12 71
12 66
12 65
12 60
12 57
12 53
12 49
12 47
12 44
12 42
12 41
12 37
12 36
12 35
12 34
12 30
12 20
12 17
12 16
12 14
12 13
11 88
11 86
11 81
11 79
11 77
11 76
11 75
11 62
11 57
11 56
11 54
11 51
11 50
11 47
11 44
11 41
11 39
11 30
11 28
11 27
11 26
11 25
11 24
11 20
11 19
11 13
10 88
10 82
10 81
10 78
10 77
10 76
10 75
10 74
10 72
10 70
10 68
10 67
10 57
10 56
10 55
10 54
10 53
10 52
10 47
10 43
10 26
10 21
10 19
10 14
9 88
9 86
9 83
9 80
9 78
9 77
9 72
9 70
9 67
9 64
9 61
9 56
9 53
9 52
9 49
9 48
9 47
9 44
9 41
9 35
9 33
9 29
9 28
9 27
9 22
9 20
9 18
9 14
8 88
8 84
8 78
8 77
8 70
8 66
8 64
8 61
8 59
8 49
8 47
8 39
8 37
8 36
8 34
8 33
8 30
8 28
8 27
8 25
8 23
8 21
8 19
8 18
8 16
7 90
7 87
7 78
7 75
7 70
7 61
7 60
7 53
7 49
7 46
7 39
7 35
7 30
7 27
7 25
7 22
7 18
6 87
6 84
6 78
6 77
6 76
6 73
6 72
6 68
6 67
6 65
6 61
6 58
6 55
6 53
6 52
6 46
6 40
2 21
2 20
1 90
1 89
1 88
1 87
1 86
1 85
1 84
1 83
1 82
1 81
1 80
1 79
1 78
1 77
1 76
1 75
1 74
1 73
1 72
1 71
1 70
1 69
1 68
1 67
1 66
1 65
1 64
1 63
1 62
1 61
1 60
1 59
1 58
1 57
1 56
1 55
1 54
1 53
1 52
1 51
1 26
1 25
1 24
1 23
1 22
1 21
1 20
1 19
1 27
1 28
1 29
1 30
1 31
1 32
1 33
1 34
1 35
1 36
1 37
1 38
1 39
1 40
1 41
1 42
1 43
1 44
1 45
1 46
1 47
1 48
1 49
1 50
2 28
2 30
2 32
2 33
2 34
2 35
2 37
2 41
2 42
2 43
2 57
2 58
2 61
2 73
2 80
2 81
2 83
3 20
3 21
3 26
3 27
3 29
3 32
3 33
3 46
3 56
3 74
3 76
3 77
3 78
3 80
3 82
3 84
3 87
4 18
4 20
4 25
4 38
4 39
4 42
4 45
4 47
4 48
4 49
4 50
4 56
4 58
4 59
4 63
4 64
4 66
4 71
4 74
4 78
4 79
4 83
4 84
4 88
5 22
5 29
5 31
5 34
5 38
5 44
5 47
5 53
5 55
5 59
5 63
5 68
5 70
5 71
5 72
5 76
5 77
5 78
5 82
6 16
6 21
6 22
6 23
6 31
6 32
6 39
13 63
13 64
13 65
13 67
13 68
13 72
13 73
13 74
13 75
13 78
13 80
13 84
13 86
13 90
14 19
14 21
14 24
14 28
14 34
14 40
14 43
14 46
14 52
14 53
14 59
14 63
14 66
14 67
14 70
14 72
14 74
14 75
14 79
14 81
14 86
14 90
15 16
15 17
15 21
15 25
15 33
15 34
15 36
15 37
15 39
15 51
15 57
15 58
15 64
15 69
15 72
15 73
15 77
15 79
15 82
15 85
15 88
16 20
16 21
16 24
16 30
16 33
16 37
16 39
16 41
16 44
16 48
16 50
16 54
16 55
16 62
16 69
16 70
16 76
16 77
16 80
16 83
16 87
16 89
17 18
17 19
17 25
17 26
17 31
17 47
17 52
17 53
17 55
17 60
17 62
17 72
17 73
17 77
17 86
17 87
17 88
18 19
18 23
18 26
18 31
18 39
18 40
18 43
18 50
18 58
18 62
18 66
18 70
18 74
18 78
18 80
18 81
18 88
18 90
19 21
19 22
19 24
19 25
19 27
19 28
19 29
19 30
19 33
19 36
19 41
19 42
19 43
19 45
19 46
19 48
19 50
19 54
19 56
19 59
19 75
19 79
19 82
19 86
19 89
20 21
20 27
20 28
20 29
20 31
20 33
20 36
20 44
20 48
20 49
20 50
20 51
20 52
20 53
20 54
20 55
20 61
20 62
20 69
20 70
20 72
20 73
20 78
20 81
20 90
21 24
21 26
21 30
21 34
21 44
21 46
21 49
21 64
21 67
21 68
21 71
21 75
21 78
21 79
21 81
21 82
21 85
21 86
21 89
22 23
22 27
22 30
22 31
22 32
22 35
22 37
22 40
22 43
22 44
22 65
22 68
22 70
22 78
22 79
22 82
22 83
22 86
22 87
22 89
22 90
23 26
23 31
23 35
23 38
23 45
23 47
23 51
23 65
23 68
23 73
23 90
24 29
24 30
24 36
24 39
24 42
24 44
24 46
24 47
24 57
24 63
24 67
24 68
24 72
24 73
24 75
24 77
24 83
24 86
25 27
25 28
25 35
25 41
25 53
25 55
25 56
25 58
25 60
25 62
25 63
25 66
25 68
25 72
25 76
25 80
25 82
26 30
26 31
26 32
26 38
26 41
26 42
26 44
26 48
26 51
26 57
26 59
26 62
26 64
26 65
26 70
26 73
26 74
26 76
26 82
26 85
26 87
27 29
27 30
27 33
27 38
27 39
27 40
27 42
27 46
27 53
27 54
27 55
27 56
27 62
27 64
27 68
27 70
27 75
27 78
27 79
27 90
28 32
28 33
28 34
28 38
28 39
28 42
28 43
28 47
28 50
28 51
28 52
28 53
28 54
28 56
28 59
28 60
28 61
28 71
28 72
28 75
28 76
28 77
28 80
28 89
29 30
29 32
29 33
29 35
29 37
29 45
29 46
29 49
29 58
29 60
29 62
29 65
29 66
29 76
29 79
29 88
29 90
30 31
30 32
30 35
30 39
30 41
30 42
30 43
30 44
30 45
30 50
30 54
30 60
30 63
30 68
30 74
30 79
30 80
30 82
30 84
30 85
30 89
31 34
31 36
31 41
31 49
31 51
31 55
31 61
31 66
31 81
32 33
32 37
32 39
32 42
32 43
32 45
32 47
32 48
32 50
32 53
32 57
32 59
32 64
32 70
32 71
32 72
32 76
32 77
32 79
32 85
32 88
33 39
3 14
1 13
0
