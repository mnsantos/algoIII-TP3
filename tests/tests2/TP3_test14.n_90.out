242 7 1 20 54 61 70 72 90
